api/repos.go

```diff
@@ -414,6 +416,145 @@ func apiReposPackageFromDir(c *gin.Context) {
 	})
 }
 
+// POST /repos/:name/copy/:src/:file
+func apiReposCopyPackage(c *gin.Context) {
+	dstRepoName := c.Request.URL.Query().Get("name")
+	srcRepoName := c.Request.URL.Query().Get("name")
```

Comment:
name twice ?

Reply:
Yeah and this was a bad idea 😅, we went with `c.Params.ByName(key)` which is actually working. I did a few manual tests which seem to indicate this work. To complete this and cover any future modification of this API, @an0rak-dev will propose a series of tests that goes with this API.

Reply:
awesome ! thanks for your work, guys :+1: